src/main/java/org/eclipse/tractusx/puris/backend/common/api/logic/dto/SuccessfullRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package org.eclipse.tractusx.puris.backend.common.api.logic.dto;
```

Comment:
Please add a license header to every file.

Reply:
Added. Checked other files, too.